### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreevlex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Avoid mixing versioned and git crates like this:
 ```toml
 # This will not compile
 [dependencies]
-gdk = "0.2"
+gtk = "0.2"
```
Comment (GuillaumeGomez) [inline README.md]: Seems weird to have gtk twice. What are you trying to do?

Comment (andreevlex) [reply README.md]: This example describes that the versions do not need to be mixed, but two different packages are specified. Do to was so as written above)

Comment (EPashkin) [reply README.md]: @andreevlex By "version" we meant package and git

Comment (sdroege) [reply README.md]: But that's exactly what @andreevlex's change is fixing here? :)

Comment (EPashkin) [reply README.md]: @sdroege IMHO including one crate from both sources is unsound in any project,
and old example show "avoid mixing versioned and git crates like this:"
IMHO "versioned" better changed to "packaged" or "published"

**Pull request closed without merging**